### DISCUSSION TITLE
How do I generate/export the slides as html?

### DISCUSSION THREAD

**Discussion by LSYS** [slidevjs/slidev#824] [Q&A]
How do I generate/export the slides as an HTML file? Like what can be done using vanilla `markdown` + `pandoc` or `revealjs`.

I have already looked into the docs (https://sli.dev/guide/exporting.html + https://sli.dev/guide/hosting.html#build-single-page-applications-spa) but I could only seem to generate an empty `index.html`.

**Comment by RoyiAvital** [comment]
I guess we can hope for an effort to add `sli.dev` as an output of Pandoc.

**Reply by george-gca** [reply]
I was able to export and host my slides online on GitHub Pages, but not run it locally as an html file.
